src/main/scala/devices/debug/Debug.scala

```diff
@@ -419,6 +428,32 @@ class TLDebugModuleOuter(device: Device)(implicit p: Parameters) extends LazyMod
       }
     }
 
+    //--------------------------------------------------------------
+    // Halt-on-reset
+    //  hrmaskReg is current set of harts that should halt-on-reset
+    //    Reset state is all zeroes
+    //    Bits are set by writing 1 to DMCONTROL.setresethaltreq
+    //    Bits are cleared by writing 1 to DMCONTROL.clrresethaltreq
+    //    Spec says if both are 1, then clrresethaltreq is executed
+    //--------------------------------------------------------------
```

Comment:
I don't recall the spec for this, but basically this bit remains set even if the core DOES halt on reset, right?